development-management/development_practice.rst

```diff
@@ -0,0 +1,115 @@
+PFS Software Development Practice
```

Comment:
Practice->Practices, as this document covers a number of practices that developers are encouraged to follow. The filename should also be updated accordingly.